common/mixnode-common/src/rtt_measurement/mod.rs

```diff
@@ -33,7 +33,7 @@ pub(crate) mod packet;
 pub(crate) mod sender;
 
 // TODO: MUST BE UPDATED BEFORE ACTUAL RELEASE!!
-pub const MINIMUM_NODE_VERSION: &str = "0.10.1";
+pub const MINIMUM_NODE_VERSION: &str = "0.10.1-dev";
```

Comment:
Why are we putting the dev suffix for the version in here? It's not like we would want to keep this later on and in semver 0.0.0-foo < 0.0.0 anyway so we're all fine

Reply:
Heheh I was under the impression that you'd explicitly asked for that - I'll just knock it out again.

Reply:
Actually now I remember why - if you don't update it, trying to `run` the node fails with: 

```
 2021-05-19T12:56:39.024Z ERROR nym_mixnode::node           > You seem to have not updated your mixnode configuration file - please run `upgrade` before attempting again
```